cmd/coattails/main.go

```diff
@@ -42,18 +44,23 @@ func initDeps() time.Duration {
 	flag.StringVar(&pgPwd, "pg-pwd", "bdc", "postgresql password")
 	flag.StringVar(&pgDb, "pg-db", "wardrobe", "postgresql db")
 	flag.StringVar(&cacheHost, "redis-host", "localhost", "redis host")
+	flag.BoolVar(&debugNoDb, "debug-nodb", false, "debug setting")
 	flag.Parse()
 	// Initialize singleton instances after parsing flag
 	sundress.InitSecret()
-	wardrobe.InitDB(fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s sslmode=disable",
-		pgHost, pgPort, pgUser, pgPwd, pgDb))
-	wardrobe.InitCache(cacheHost)
+	if !debugNoDb {
```

Comment:
If we're starting the server without initializing db or cache, I assume the endpoints that use them would segfault if we call them? I think it's fine to start the server in this state, but we should at least throw a fat warning on start so we know - also rename to `run-without-deps` or something? since we're also not using the cache

Reply:
resolved.